modules/compliance-anatomy.adoc

```diff
@@ -299,14 +300,14 @@ The remediation loop ends once the rendered machine config is updated, if needed
 
 [source,terminal]
 ----
-$ oc annotate compliancescans/<scan_name> compliance.openshift.io/rescan=
+$ oc annotate -n openshift-compliance compliancescans/<rhcos4-e8-worker compliance.openshift.io/rescan=
```

Comment:
Remove the '<'?
Should be:
`oc annotate -n openshift-compliance compliancescans/rhcos4-e8-worker compliance.openshift.io/rescan=`